### DISCUSSION TITLE
how limit predict value between 0~1 or 1~100

### DISCUSSION THREAD

**Discussion by jaffe-fly** [ourownstory/neural_prophet#537] [Q&A - forecasting best practices] [answered]
when i predict  a value beyond the max value
so how to set the predict value between 1~100?

**Comment by ourownstory** [comment, marked as answer]
Hi @jaffe-fly 
As of now, there is no option to limit, though logistic growth (which does more than just cap) may be added in the future.

For most applications, you could simply post-process the forecasts to cap them - would that be a viable option for you?

**Reply by jaffe-fly** [reply]
What I thought is that if it is greater than max, it is directly equal to max. I don't know if there is a better way

**Reply by ourownstory** [reply]
yeah, you could simply do something like 
(untested pseudocode)
```python
df['yhat1'] = np.maximum(0, np.minimum(1, df['yhat1'].values))
```
